nirc_ehr/resources/queries/study/cases.js

```diff
@@ -15,21 +15,29 @@ function onInit(event, helper) {
 
 EHR.Server.TriggerManager.registerHandlerForQuery(EHR.Server.TriggerManager.Events.BEFORE_UPSERT, 'study', 'cases', function(helper, errors, row, oldRow){
     if (!helper.isETL()) {
+        var error = false;
         if (row.enddate) {
             if (!row.closeRemark) {
                 EHR.Server.Utils.addError(errors, 'closeRemark', 'Close remark required when closing a case.', 'ERROR');
+                error = true;
             }
 
             if(!triggerHelper.canCloseCase(row.category)) {
                 EHR.Server.Utils.addError(errors, 'enddate', 'Veterinarian permission required to close a case.', 'ERROR');
+                error = true;
             }
         }
 
+        if (!row.performedby) {
+            EHR.Server.Utils.addError(errors, 'performedby', 'Opened by is a required field.', 'ERROR');
```

Comment:
"Opened by is a required field" --should it be "'Performed By' is a required field"?